region/norfair/crocomire/Post Crocomire Jump Room.json

```diff
@@ -799,15 +799,15 @@
         "canUseFrozenEnemies",
         "canManipulateMellas",
         "canBeVeryPatient",
```

Comment:
```suggestion
        {"or": [
          {"and": [
            "canBePatient",
            "canWalljump"
          ]},
          "canBeVeryPatient"
        ]},
```

Reply:
120frames door to item with 1 walljump.
Watch it climb from the ground and once it's lowest point is around jump height, climb aboard and MellaIceClimb and that puts you close enough to walljump to the ledge.  A Springball jump probably works _quickly_ if done this way too.